src/components/post-comment.jsx

```diff
@@ -72,6 +72,9 @@ export default class PostComment extends React.Component {
     if (shouldClearText) {
       this.setState({editText: ''});
     }
+    if (this.props.editText !== newProps.editText) {
```

Comment:
`this.state.editText` has nothing to do with Redux state, it's a state you change on the next line. So my question is: why you are checking `this.props.editText` instead of `this.state.editText`?